### PR TITLE
Implement is unique algorithm in Go

### PR DESCRIPTION
**Pull request by Lumexralph**

### CODE REVIEW

File: go/array_string/unique/is_unique.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+// and space complexity amortized O(1) - O(n)
+// if the whole string is an ASCII, we'll be dealing
+// with 128 characters and if it is Unicode, it will be more.
+package unique
+
+func isUnique(words string) bool {
+	visited := make(map[rune]bool)
+
+	for _, char := range words {
+		// if we have seen the character before
```
Comment (ARR4N) [inline go/array_string/unique/is_unique.go]: I like to think of comments as explaining "why" but never "what". If you have to explain what you are doing then the code probably needs to be fixed up (e.g. rename variables or even totally refactored). If you have a reason for making a decision then it usually can't be communicated in the code itself so should be a comment.

TL;DR this comment isn't necessary

**Review by ARR4N — COMMENTED** [review]
Looks good to me!

Are you solving these yourself, or implementing the existing solutions just in Go? I think it's best to ignore the solutions entirely and just give it a shot
